src/stories/Breadcrumb/BreadcrumbMeta.ts

```diff
@@ -7,19 +13,4 @@ export const meta = {
       },
     },
   },
-  argTypes: {
```

Comment:
- Add elements control arg as a "control" in category field
- Add default slot as "slots" in category field (cf. accordionMeta or other components with slot unnamed